diff --git a/dist/server/static-file-cache.js b/dist/server/static-file-cache.js
index ba3bec2bc1156662581cbabc0ab116728cbf2968..8050f372a5d9855207cabd87af6044d49b0893a9 100644
--- a/dist/server/static-file-cache.js
+++ b/dist/server/static-file-cache.js
@@ -204,7 +204,7 @@ async function* walkFilesWithStats(dir, base = dir) {
 		const batch = files.slice(i, i + STAT_BATCH_SIZE);
 		const stats = await Promise.all(batch.map((f) => fsp.stat(f)));
 		for (let j = 0; j < batch.length; j++) yield {
-			relativePath: path.relative(base, batch[j]),
+			relativePath: path.relative(base, batch[j]).replaceAll("\\", "/"),
 			fullPath: batch[j],
 			stat: {
 				size: stats[j].size,
